post-scan-actions/gcp-python-slack-notification/serverless.yml

```diff
@@ -0,0 +1,43 @@
+service: cloudone-fss-gcp-slack-notifications
+
+custom:
+  defaultStage: prod
+
+params:
+  prod:
```

Comment:
I'm wondering is there any use case we'll need to add other stages in this plugin? Some properties, like `EVENT_TYPE`, does not likely to change in different stages in my opinion.
If you intend to define some variables, you can use custom section like this: https://www.serverless.com/framework/docs/providers/aws/guide/variables#reference-properties-in-serverlessyml

Reply:
I was thinking about moving `EVENT_TYPE` to `params.defaults` section but then I did not want to have more than one place for users/customers to configure this tool before deployment so I put them all together. I have used custom variables to set the defaultStage to `prod` for now.

Reply:
I'm still wondering why we need to define a stage in the template and specify a stage here? For defining variables, you could use `custom` session and refer them as `${self:custom.XXX}`. For example:
```yaml
custom:
  eventType: providers/cloud.pubsub/eventTypes/topic.publish
  deploymentRegion: ${param:DEPLOYMENT_REGION, 'us-east1'}  # just an example to set a default value if CLI parameter is not given

provider:
  region: ${self:custom.deploymentRegion}
```

Reply:
I am using a `defaultStage` and if a stage was not passed as a CLI param using ${opt:stage}, example: `stage: ${opt:stage, self:custom.defaultStage}`

![Screen Shot 2022-08-08 at 11 01 57](https://user-images.githubusercontent.com/64860010/183449479-2a7b9182-0795-45d0-b5e6-22cb4fbac64a.jpg)